x/callback/keeper/fees.go

```diff
@@ -0,0 +1,47 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/archway-network/archway/x/callback/types"
+)
+
+func (k Keeper) EstimateCallbackFees(request *types.QueryEstimateCallbackFeesRequest, ctx sdk.Context) (sdk.DecCoin, sdk.DecCoin, sdk.DecCoin, error) {
```

Comment:
we can swap request with `estimatedHeight`, so gRPC specific APIs do not leak outside of grpc_server